dash-pipeline/tests/saithrift/ptf/vnet/test_saithrift_vnet.py

```diff
@@ -94,9 +94,10 @@ def runTest(self):
         # TODO expected packet might be different
         self.udp_pkt_exp = self.udp_pkt
         print("\nSending packet...", self.udp_pkt.__repr__())
-        send_packet(self, 0, self.udp_pkt)
-        print("\nVerifying packet...", self.udp_pkt_exp.__repr__())
-        verify_packet(self, self.udp_pkt_exp, 0)
+        #send_packet(self, 0, self.udp_pkt)
```

Comment:
I'm not in favor of removing any test of pipeline continuity. I'd much prefer we configure the pipeline even with basic wildcard match rules, keep the echo test, then have a test which proves no packets make it without config. (I understand this echo test was relying on incorrect behavior but I don't want to lose some verification.)

Reply:
I am going to be creating a follow up PR immediately to convert this to an actual dash vxlan outbound packet test. Already have all the changes ready. Just waiting for the ptf vxlan fix.

Reply:
Changed this to a negative test to verify invalid packet drop

Reply:
OK.